de.fu_berlin.inf.dpp.core/build.gradle

```diff
@@ -0,0 +1,69 @@
+dependencies {
+  releaseDep 'commons-codec:commons-codec:1.3'
+  releaseDep 'commons-io:commons-io:2.0.1'
+  releaseDep 'commons-lang:commons-lang:2.4'
+  releaseDep 'javax.jmdns:jmdns:3.4.1'
+  releaseDep 'org.picocontainer:picocontainer:2.11.2'
+  releaseDep 'org.igniterealtime.smack:smack:3.4.1'
```

Comment:
We have a modified version of smack, this time the patched code is not inside the jar files but overridden via the bundle classpath

Reply:
I will verify that the patches are loaded before the jars, but I think I already checked this when I added the `/patches` directory as source directory.